vendor/src/github.com/docker/libnetwork/network.go

```diff
@@ -206,6 +209,13 @@ func (n *network) ID() string {
 	return n.id
 }
 
+func (n *network) Created() time.Time {
+	n.Lock()
```

Comment:
In current libnetwork design, read should block while write (e.g. `CopyTo()`) is in progress 
https://github.com/docker/libnetwork/blob/8dd6270f208ce449ec69140a2b7103d8190189cc/network.go#L1442-L1542

I agree we can optimize locks, but it should be another PR